docs/python/programming/math/vector.rst

```diff
@@ -549,6 +549,33 @@ Vector
     .. seealso::
         :meth:`Vector.record`, :meth:`Vector.play_remove`
     
+    Example of playing into an Iclamp for varying current:
+
+        .. code-block::
+            python
+        
+        from neuron import h
+        import pylab as plt, numpy as np
+        plt.ion()
```

Comment:
I'd do a `plt.show()` at the end instead, but I think that's personal preference.